cookbooks/aws-parallelcluster-slurm/libraries/helpers.rb

```diff
@@ -104,9 +104,11 @@ def share_munge_head_node
     group 'root'
     code <<-HEAD_SHARE_MUNGE_KEY
       set -e
-      mkdir -p /home/#{node['cluster']['cluster_user']}/.munge
+      mkdir -p #{node['cluster']['shared_dir']}/.munge
```

Comment:
We should also share it with the login nodes.

Reply:
Done!